styles.css

```diff
@@ -0,0 +1,949 @@
+    width: 400px;
+  }
+}
+@media (max-width: 600px) {
+  main {
+    max-width: 100%;
+  }
+  .header {
+    padding: 0px;
+  }
+  .navbar > ul > li > a {
+    padding: 0px 15px;
+  }
+  .home-image {
+    width: 300px;
+  }
+  .blob-image {
+    width: 100px;
+    top: 300px;
+    left: 0px;
+  }
+}
+@media (max-width: 550px) {
+  .home-image {
+    width: 270px;
+  }
+  .blob-image {
+    width: 100px;
+    top: 300px;
+  }
+}
+@media (max-width: 400px) {
+  .navbar > ul > li > a {
+    padding: 0px 10px;
+  }
+  .home-image {
+    width: 200px;
+  }
+  .blob-image {
+    width: 100px;
+    top: 300px;
+  }
+}
+
+/* -------------------- sec2 ------------------------- */
+
+.sec2 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  padding: 0px var(--padding-x);
+  padding-top: 80px;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 20px;
+  flex-direction: column;
+  width: auto;
+
+  transform: translateX(-100%);
+  opacity: 0;
+  transition: transform 0.5s ease-out, opacity 0.5s ease-out;
+}
+
+.personal-image {
+  width: 570px;
+  height: auto;
+
+  transform: translateX(100%);
+  opacity: 0;
+  transition: transform 0.5s ease-out, opacity 0.5s ease-out;
+}
+
+.about-me-title {
+  font-size: 40px;
+  font-weight: semibold;
+  color: var(--color-orange-dark);
+}
+
+.about-me-text {
+  color: var(--color-gray-dark);
+  font-size: 32px;
+  max-width: 500px;
+}
+
+@media (max-width: 1200px) {
+  .sec2 {
+    flex-direction: column;
+    padding: 0px var(--padding-x);
+    padding-top: 80px;
+  }
+  .about-container {
+    gap: 15px;
+  }
+  .about-me-title {
+    font-size: 36px;
+  }
+  .about-me-text {
+    font-size: 28px;
+  }
+  .personal-image {
+    width: 450px;
+  }
+}
+
+@media (max-width: 600px) {
+  .sec2 {
+    padding: 0px var(--padding-x);
+    padding-top: 80px;
+  }
+  .about-container {
+    gap: 10px;
+  }
+  .about-me-title {
+    font-size: 30px;
+  }
+  .about-me-text {
+    font-size: 24px;
+    max-width: 400px;
+  }
+  .personal-image {
+    width: 350px;
+  }
+}
+
+@media (max-width: 550px) {
+  .sec2 {
+    padding: 0px var(--padding-x);
+    padding-top: 80px;
+  }
+  .about-me-title {
+    font-size: 26px;
+  }
+  .about-me-text {
+    font-size: 22px;
+    max-width: 350px;
+  }
+  .personal-image {
+    width: 300px;
+  }
+}
+
+@media (max-width: 400px) {
+  .sec2 {
+    padding: 0px var(--padding-x);
+    padding-top: 80px;
+  }
+  .about-container {
+    gap: 5px;
+  }
+  .about-me-title {
+    font-size: 24px;
+  }
+  .about-me-text {
+    font-size: 18px;
+    max-width: 300px;
+  }
+  .personal-image {
+    width: 250px;
+  }
+}
+
+/* -------------------- sec3 ------------------------- */
+.sec3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 100px;
+  padding: 0px 150px;
+  padding-top: 80px;
```

Comment:
Use  `padding: 80px 150px 0px 150px ;` instead

Reply:
fixed in all places